lldb/source/Interpreter/CommandInterpreter.cpp

```diff
@@ -2078,6 +2078,11 @@ bool CommandInterpreter::HandleCommand(const char *command_line,
 
     ElapsedTime elapsed(execute_time);
     cmd_obj->SetOriginalCommandString(real_original_command_string);
+    pos = real_original_command_string.rfind(remainder);
```

Comment:
IIUC, the indent gets incremented twice here.  The first time is here where you're just conveniently setting it to "past the command name" since commands don't get that part.  Then later if there's an error you bump the indent to the actual error.  That seems sufficiently tricky to warrant a comment?